pytorch/training/buildspec.yml

```diff
@@ -1 +1 @@
-buildspec_pointer: buildspec-2-3-sm.yml
```

Comment:
note: you can still use the buildspec overrides in the toml file to specify buildspec, to minimize # of files where you make temp changes. But if this is simpler/more intuitive and you prefer to do it this way, that's OK too, just make sure to revert before merge